grafana/dashboards/openfga.json

```diff
@@ -0,0 +1,1161 @@
+{
```

Comment:
Does it make sense to put all the observability stuff like this and `promethus` in an `observability` folder (or `telemetry` or something)? Also the `otel-collector-config.yaml` file.

Reply:
Probably 😄 .  Let's put it in a `telemetry` folder 👍

Reply:
Put the telemetry configuration manifests in the top level `telemetry` folder.